load.go

```diff
@@ -161,6 +161,16 @@ func (s *Spec) SubstituteArgs(env map[string]string, opts ...SubstituteOpt) erro
 		s.Sources[name] = src
 	}
 
+	for k, v := range s.Patches {
```

Comment:
Can we change these loop variables to be slightly more descriptive, maybe `src, patchList := range s.Patches`

Reply:
Done.